test/benchmarks/src/main/scala/scala/collection/mutable/OpenHashMapRunner.scala

```diff
@@ -0,0 +1,105 @@
+package scala.collection.mutable
+
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileOutputStream
+import java.io.OutputStreamWriter
+import java.io.PrintWriter
+import scala.collection.JavaConversions
+import scala.language.existentials
```

Comment:
This import is kind of lost here, which is contrary to the point of requiring the import in the first place.  Can you move it to the top?